courses/free-courses-ne.md

```diff
@@ -7,12 +7,15 @@
 * [SQL](#sql)
 * [C](#c)
 * [WordPress](#wordpress)
+* [PHP](#php)
 
 
 ### JavaScript
 
 * [JavaScript](https://www.youtube.com/playlist?list=PLckS_N3kOwFH-GCqCd6i-vPo-Z75DcOnc) - EverydayKarma 🇳🇵
 
+* [JavaScript Basic Tutorial In Nepali](https://youtu.be/_4e_Jyjmudg?si=MnMhuhWOzzXiGWIi) - Technology Channel
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
no shortened links!